S2VX.Game/Story/Note/HitMarkers.cs

```diff
@@ -0,0 +1,49 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osuTK;
+using osuTK.Graphics;
+using System.Collections.Generic;
+
+namespace S2VX.Game.Story.Note {
+    public class HitMarkers : CompositeDrawable {
```

Comment:
I have some general comments after some playtesting:
* Miss markers don't show up
* Let hit markers stay longer, osu! has a pretty long window for this I think
* Change hit marker colors to be more contrasting, e.g. blue, orange maybe
* Remove hit marker for perfect
* Remove color changes on cursor

Reply:
I agree with all of these points. Right now it's pretty hard to tell if I'm late or early just by the color since perfect is yellow, and  slightly late&early are different shades of yellow. Having big contrast between late & early + the current shade changing would be helpful

Reply:
- Miss markers seem to be working for me
- I'll change to 800ms and see how it is
- I've changed early to Cyan and late to Magenta
- Removed perfect
- Removed cursor color changes